x-pack/plugins/apm/public/components/app/diagnostics/apm_documents_tab.tsx

```diff
@@ -57,36 +53,40 @@ export function DiagnosticsApmDocuments() {
     {
       name: 'Name',
       field: 'name',
-      width: '40%',
+      width: '30%',
     },
     {
       name: 'Doc count',
       field: 'docCount',
-      render: (_, { docCount }) => asInteger(docCount),
+      render: (_, { docCount }) => (
+        <EuiToolTip content={`${asInteger(docCount)} docs`}>
+          <div style={{ cursor: 'pointer' }}>{asBigNumber(docCount)}</div>
+        </EuiToolTip>
+      ),
       sortable: true,
     },
     {
       name: '1m',
       field: 'intervals.1m',
       render: (_, { intervals }) => {
-        const docCount = intervals?.['1m'];
-        return formatDocCount(docCount);
+        const interval = intervals?.['1m'];
+        return <IntervalDocCount interval={interval} />;
```

Comment:
Really useful to show both doc and event count but the formatting is a bit confusing. 

For example, 
`66 (2k events)` vs `66 docs / 1,600 events`

What do you think of the following? 
`~ 66 (2k events)` vs `65,665 docs (1,600 events)`



Also, why did you decide to break down only for the rollups and not the Doc count (total doc counts)